scripts/system/create/edit.js

```diff
@@ -2613,6 +2616,46 @@ var PropertiesTool = function (opts) {
                         Entities.reloadServerScripts(selectionManager.selections[i]);
                     }
                 }
+            } else if (data.action === "copyPosition") {
+                selectionManager.saveProperties();
+                properties = selectionManager.savedProperties[selectionManager.selections[0]];
+                copiedPosition = properties.position;
+                Window.copyToClipboard(JSON.stringify(copiedPosition));
+                print("COPIED: " + JSON.stringify(copiedPosition));
```

Comment:
It could be good to safeguard this code (and that for for rotation) with a check that the number of selections is 1 - even though the buttons are (should be) disabled if the number of selections == 0 or > 1.

Reply:
I added safeguard on the 4 actions (Copy and Paste) to not trust only the UI.